src/__Private/LinterCLI.php

```diff
@@ -156,6 +156,19 @@ private function lintPath(
         return 1;
       }
     } else {
+      foreach ($roots as $root) {
+        $path = \realpath($root);
+        if (\is_dir($path)) {
+          $config_file = $path.'/hhast-lint.json';
+          if (\file_exists($config_file)) {
+            \fwrite(
+              \STDERR,
+              "Warning: PATH arguments contain a hhast-lint.json\n",
+            );
+            return 1;
```

Comment:
As this is a warning, not an error, it probably shouldn't exit non-zero, and should probably still run the lints

Reply:
Should this go to STDOUT then?